librz/core/cmd/cmd_print.c

```diff
@@ -844,6 +844,36 @@ static const ut32 colormap[256] = {
 	0xffffff,
 };
 
+static void incAlphaBuffer(ut8 *buf, int bufsz) {
```

Comment:
what are these?

Reply:
These are some functions that I needed to keep such that the build passes, I will refactor them in the end.